h2o-genmodel/src/main/java/hex/genmodel/algos/drf/DrfMojoModel.java

```diff
@@ -56,4 +58,8 @@ public DrfMojoModel(String[] columns, String[][] domains, String responseColumn)
         return score0(row, 0.0, preds);
     }
 
+    @Override
```

Comment:
this should be defined on SharedTreeMojoModel, not in each subclass, this makes adding new subclasses harder